### DISCUSSION TITLE
Which instruction.txt should we use?

### DISCUSSION THREAD

**Discussion by chen-rn** [machinewrapped/llm-subtrans#52] [General]
Hey! I saw that you added a few more instruction.txts in the recent days, one for chinese, and another for OCR.

Does these instruction.txts greatly effect translation quality?

**Comment by machinewrapped** [comment]
Sorry for the delay, been out of town :-)

The default instructions.txt is a good choice for most circumstances - it's certainly the most thoroughly tested. "Chinese + English" and "OCR Errors" instructions were for some source subtitles which had more specific requirements, to make it clear what was expected.

The instructions definitely affect the quality of the translation. It's especially important to provide examples, as GPT will do its best to mimic the form and style. If there's a particular task you want it to perform regularly, e.g. certain language pairs, it's probably worth creating custom instructions with examples of "what good looks like".
